cmd/kusk/cmd/generate.go

```diff
@@ -169,9 +188,62 @@ var generateCmd = &cobra.Command{
 			fmt.Fprintln(os.Stderr, err)
 			os.Exit(1)
 		}
+
+		if !apply {
```

Comment:
I was under the impression that we weren't going to add auto apply from the CLI since that's what `kubectl` is for?